src/platforms/common/server/buffer_from_wl_shm.cpp

```diff
@@ -390,7 +391,7 @@ class WlShmBuffer :
 
             auto size() const -> mir::geometry::Size override
             {
-                return parent->size();
+                return parent->ShmBuffer::size();
```

Comment:
Explicitly calling the `ShmBuffer` implementation of `size()` here is unnecessary, right? Both `parent->ShmBuffer::size()` and `parent->RWMappableBuffer::size()` call the same (necessarily explicitly implemented, because `RWMappableBuffer` is abstract) `WlShmBuffer::size()` implementation.